### DISCUSSION TITLE
Missing values in Testcase 5 output files

### DISCUSSION THREAD

**Discussion by srikar-iist** [NASA-LIS/LISF#1412] [Q&A] [answered]
Hello,
Dear @karsenau @dmocko 
I tried to run through testcase 5 -in LIS 7.2 version due to my technical constraints - (my max snow option was ./ INPUT/LS_PARAMETERS/noah_2dparms/maxsnoalb.asc of NCEP_Native source), and it all went smoothly and the log file seems to be normal as it shows below.

Writing to LDT output directory: ./
Writing CDF file .//cdf_smapobs.nc
Writing CDF domain file .//cdf_smapobs_domain.nc
Successfully wrote CDF file .//cdf_smapobs.nc
Successfully wrote CDF file .//cdf_smapobs_domain.nc
Finished LDT run

However, cdf_smapobs_domain.nc variables are all filled with missing values. There aint any problem with observational data, as i have also checked for SMOPS-ASCAT and the same issue occurs.

I have already check issues #1063 and #1064 - this may not apply to my case as my predecessors have installed the software using HDF support - 4,5, and EOS. Kindly advice

**Comment by dmocko** [comment, marked as answer]
Hi @srikar-iist 

Please use only the `.gr2` files for this test.  See discussion #1409 please.
